apps/wallet-dashboard/lib/utils/vesting/vesting.spec.ts

```diff
@@ -11,14 +11,15 @@ import {
     mockedSupplyIncreaseVestingTimelockedObjectsWithDynamicDate,
 } from '../../constants';
 import { SupplyIncreaseUserType, SupplyIncreaseVestingPayout } from '../../interfaces';
-import { formatDelegatedTimelockedStake, isTimelockedObject } from '../timelock';
+import { isTimelockedObject } from '../timelock';
 import {
     getVestingOverview,
     buildSupplyIncreaseVestingSchedule as buildVestingPortfolio,
     getLatestOrEarliestSupplyIncreaseVestingPayout,
     getSupplyIncreaseVestingPayoutsCount,
     getSupplyIncreaseVestingUserType,
 } from './vesting';
+import { formatDelegatedTimelockedStake } from '@iota/core/utils/formatDelegatedTimelockedStake';
```

Comment:
`formatDelegatedTimelockedStake` and `MILLISECONDS_PER_HOUR` can be imported just from @iota/core